src/AccessibilityInsights.CommonUxComponents/Controls/FabricIconControl.xaml

```diff
@@ -0,0 +1,14 @@
+<!-- Copyright (c) Microsoft. All rights reserved.
+     Licensed under the MIT license. See LICENSE file in the project root for full license information.-->
+<UserControl x:Class="AccessibilityInsights.CommonUxComponents.Controls.FabricIconControl"
```

Comment:
There are still many XAML files that reference AccessibliityInsights.DesktopUI.Controls, presumably to use the *FabricIconControl. Will these be cleaned up in this PR or in a separate change?

Reply:
these should be cleaned up in a separate change
@jalkire